manager-bundle/src/ContaoManager/Plugin.php

```diff
@@ -334,25 +334,41 @@ private function addDefaultServerVersion(array $extensionConfigs, ContainerBuild
      *
      * @return array<string,array<string,array<string,array<string,mixed>>>>
      */
-    private function addDefaultPdoDriverOptions(array $extensionConfigs): array
+    private function addDefaultPdoDriverOptions(array $extensionConfigs, ContainerBuilder $container): array
     {
         // Do not add PDO options if the constant does not exist
         if (!\defined('PDO::MYSQL_ATTR_MULTI_STATEMENTS')) {
             return $extensionConfigs;
         }
 
+        $driver = null;
+        $url = null;
         foreach ($extensionConfigs as $extensionConfig) {
-            // Do not add PDO options if the selected driver is not pdo_mysql
-            if (isset($extensionConfig['dbal']['connections']['default']['driver']) && 'pdo_mysql' !== $extensionConfig['dbal']['connections']['default']['driver']) {
-                return $extensionConfigs;
+            if (isset($extensionConfig['dbal']['connections']['default']['driver'])) {
+                $driver = $extensionConfig['dbal']['connections']['default']['driver'];
+            }
+
+            if (isset($extensionConfig['dbal']['connections']['default']['url'])) {
+                $url = $container->resolveEnvPlaceholders($extensionConfig['dbal']['connections']['default']['url'], true);
             }
 
             // Do not add PDO options if custom options have been defined
+            // Since this is merged recursively, we don't need to check other configs
             if (isset($extensionConfig['dbal']['connections']['default']) && \array_key_exists('options', $extensionConfig['dbal']['connections']['default'])) {
```

Comment:
```suggestion
            if (\array_key_exists(\PDO::MYSQL_ATTR_MULTI_STATEMENTS, $extensionConfig['dbal']['connections']['default']['options'] ?? [])) {
```
I think we should check for the multi statements option here directly as many users now set the options to enable the strict mode.

Reply:
if any option is set, we don't add our own options. I think that already solves this?

Reply:
> if any option is set, we don't add our own options.

Which is wrong IMO because this now happens on nearly every installation because of the strict mode.

I think we should only skip our own options if the `PDO::MYSQL_ATTR_MULTI_STATEMENTS` is set.

Reply:
added in c3bd64236b7bec212f8758a0d5ef50a9915faa45